qlib/data/data.py

```diff
@@ -223,8 +223,11 @@ def convert_instruments(self, instrument):
             for _path in Path(C.get_data_path()).joinpath("instruments").glob("*.txt"):
                 _df = pd.read_csv(_path, sep="\t", names=["inst", "start_datetime", "end_datetime", "save_inst"])
                 _df_list.append(_df.iloc[:, [0, -1]])
-            df = pd.concat(_df_list, sort=False).sort_values("save_inst")
-            df = df.drop_duplicates(subset=["save_inst"], keep="first").fillna(axis=1, method="ffill")
+            df = pd.concat(_df_list, sort=False)
```

Comment:
```
dtype:  Type name or dict of column -> type, optional
      Data type for data or columns. E.g. {‘a’: np.float64, ‘b’: np.int32, ‘c’: ‘Int64’} Use str or object together with suitable na_values settings to preserve and not interpret dtype. If converters are specified, they will be applied INSTEAD of dtype conversion.
```
Will dtype be a cleaner solution?